rust/src/accuracy/polars/mod.rs

```diff
@@ -66,6 +66,7 @@ where
     lazyframe_utility(&lf, alpha)
 }
 
+#[derive(Clone)]
```

Comment:
I'd prefer not to edit the Rust in this PR, if at all possible: if the examples in the docs rely on changes here, then we should get another release out before we point people to the nightly docs. It's adding more steps.

(But if this is a change we really need, don't let me block!)

Reply:
Broke it out into a separate PR. It also killed the commit history here, unfortunately.

Reply:
Huh: Github seems to be confused. The base PR in the stack is merged, and then checked out `main`, and confirmed that this `#[derive(Clone)]` is in there... So it seems like it shouldn't be marked as a change here, as well? Which makes me wonder about how well this UI is representing other changes in this PR.

I'm going to try diffing this branch with `main` locally, and will see what that looks like.